velox/core/QueryConfig.h

```diff
@@ -710,6 +719,23 @@ class QueryConfig {
     return get<uint32_t>(kDriverCpuTimeSliceLimitMs, 0);
   }
 
+  static int32_t prefixSortMaxNormalizedKeySize(const velox::Config* config) {
```

Comment:
This API is not consistent with the rest. Please, remove. In the future, please, call out deviations from coding/API style.

Reply:
This is for HiveDataSink,  I can not get a QueryConfig, and it seems strange to create a new one.